Dz3/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Reflection.Metadata.Ecma335;
+using System.Text.RegularExpressions;
+
+namespace DZ2DZ3
+{
+    public class Program
+    {
+        public enum Gender
+        {
+            male,
+            female,
+            undefined
+        }
+        static void Main(string[] args)
+        {
+            User user1 = new User();
+            // user1.Gender = Gender.Male;
+            try
+            {
+                while (true)
+                {
+                    Console.WriteLine(" Pleace input your name");
+                    user1.FirstName = Console.ReadLine();
+
+                    if (Regex.IsMatch(user1.FirstName, "^[a-zA-Z]+$"))
+                    {
+                        user1.FirstName = user1.FirstName;
+                        break;
```

Comment:
The operator `break` use is to stop loops, why it is in the `if-else` construction?

Reply:
because if the user has entered his own name correctly, we can go further, he can't go any other way, if he just wrote a character, it's an infinite loop